CGPs/0013.md

```diff
@@ -0,0 +1,83 @@
+# CGP [0013]: Celo Core Contracts Release 1
+
+- Date: 2020-10-16
+- Author(s): @nambrot
+- Status: DRAFT
+- Governance Proposal ID #: [if submitted]
+- Date Executed: [if executed]
+
+## Overview
+
+In this governance proposal, we propose to complete the first Celo Core Contracts Upgrade. This release debuts a new [release process](https://docs.celo.org/community/release-process/smart-contracts) that culminates in this proposal. The goal of this new release process is to enable more agile changes to the Celo Core Contracts while prioritizing realiability and security of the whole Celo platform. cLabs engineers have invested significant effort in verfication tooling in the past months to allow Celo stakeholders to assert the validity of the prposal itself. See more under [Verification](#Verification)
```

Comment:
Consider adding link to [Core Contracts](https://github.com/celo-org/celo-monorepo/tree/master/packages/protocol/contracts)